plugins/sqlfluff-templater-dbt/docker/init

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+pip install --no-dependencies -e . -e plugins/sqlfluff-templater-dbt
```

Comment:
Need to do all this stuff once the container is running in order for the tests to work in Docker.